### PR TITLE
sys-fs/mtd-utils: bump to 2.1.2

### PR DESCRIPTION
**Pull request by jsmolic**
Closes: https://bugs.gentoo.org/722742
Package-Manager: Portage-2.3.99, Repoman-2.3.23
Signed-off-by: Jakov Smolic <jakov.smolic@sartura.hr>

### CODE REVIEW

**Comment by jsmolic** [comment]
If it's okay with maintaners of this package, I would add myself as co-maintainer, I am willing to take care of this package in the long run

**Comment by jsmolic** [comment]
@juippis I'm curious, can this be merged due to maintainer timeout? (at least the version bump commit).\
Bug (https://bugs.gentoo.org/722742) is opened for several months now, and I've received no feedback on this whatsoever :(

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (juippis) [inline sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:6]: Is *vcs-snapshot* really needed here?

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (juippis) [inline sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:16]: Please sort this list alphabetically.

**Review by juippis — COMMENTED** [review]
Sure, version bump looks ok, but I won't change metadata.xml yet. Please provide more contributions. You should be able to ping robbat2 via IRC or directly by e-mail in future.

This looks good, just check whether the *vcs-snapshot* is needed or not. Let me know if it is needed, I'll merge while dropping the metadata.xml commit.

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:6]: Yes if using http://git.infradead.org/mtd-utils.git repository. I found https://github.com/sigma-star/mtd-utils, I can change the ebuild to use that repo

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild]: Done

**Comment by jsmolic** [comment]
> Sure, version bump looks ok, but I won't change metadata.xml yet. Please provide more contributions. You should be able to ping robbat2 via IRC or directly by e-mail in future.

Great, thanks, I dropped the other commit

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild]: Done

File: sys-fs/mtd-utils/Manifest
Comment (juippis) [inline sys-fs/mtd-utils/Manifest:3]:
```
!!! Fetched file: mtd-utils-2.1.2.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      380333
!!! Expected: 380370
```

Whaat's up with this? Don't tell me it changes everytime it's downloaded based on timestamp or something.

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (juippis) [inline sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:6]: Sorry if I just noticed this, but *eutils* doesn't seem to be used by this ebuild at all.

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (juippis) [inline sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:46]: You could add a `DOCS` array before `src_prepare` so these'd get installed with the `default` command here.

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild]: Thanks, dropped

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild]: Done, thanks

File: sys-fs/mtd-utils/Manifest
Comment (jsmolic) [reply sys-fs/mtd-utils/Manifest:3]: Hm, not really sure, I didn't encounter this. I am using regular github tarball, can you please try again now?

Comment (juippis) [reply sys-fs/mtd-utils/Manifest:3]: Clear your distfiles, `rm Manifest` and then use `repoman manifest`. You must have downloaded the one from previous host, because...

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (juippis) [inline sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:9]: ... ```
 * ERROR: sys-fs/mtd-utils-2.1.2::testworld failed (prepare phase):
 *   The source directory '/var/tmp/portage/sys-fs/mtd-utils-2.1.2/work/mtd-utils-v2.1.2-0ef86b1' doesn't exist
```

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (juippis) [inline sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild:12]: Since you have fixing to do, could you move these to correct skel positions? I would've done that after merging but then I came across the "no such directory" error...
```
EAPI=7                                                                                              
                                                                                                    
DESCRIPTION=""                                                                                      
HOMEPAGE=""                                                                                         
SRC_URI=""                                                                                          
                                                                                                    
LICENSE=""                                                                                          
SLOT="0"                                                                                            
KEYWORDS="~amd64 ~x86"                                                                              
IUSE=""                                                                                             
                                                                                                    
DEPEND=""                                                                                           
RDEPEND="${DEPEND}"                                                                                 
BDEPEND=""           
```

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild]: Something was wrong with my distfiles, that also caused the workdir error, it should be okay now.

File: sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild
Comment (jsmolic) [reply sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild]: Done.\
Thank you for your patience and review

**Review by juippis — APPROVED** [review]
I'm not the only one who's been patient here ;)

Thanks!

**Pull request closed without merging**
